sdk/include/psdk/windowsx.h

```diff
@@ -558,4 +583,8 @@
 
 #define hmemcpy MoveMemory
 
+#ifdef __cplusplus
+}
 #endif
+
+#endif /* _INC_WINDOWSX */
```

Comment:
Probably it's worth to document with what version this file is synced now (in the `/media/doc/3rd Party Files.txt`).

Reply:
It is synced with nothing, anymore. It came from some old stuff from year 2004-2005 unchanged, except for our small (or less smaller) fixes:
https://git.reactos.org/?p=reactos.git;a=history;f=reactos/w32api/include/windowsx.h;h=460718e6409dc0a0ee6efc10f0217ddf921604c7;hb=cb4308061a829c28222d74a8a5f6e392d162663f